Lambda-School-Labs/Revitalize-FE
# Feature/status fix

Description:  status fix 
Fixes # (issue) As a General User/Project Manager/Trades Professional.. I wish I could accept my choices instead of using these drop downs that auto update and lock in the dashboard
Type of change
Please delete options that are not relevant.
 Bug fix (non-breaking change which fixes an issue)
 This change requires a documentation update
Change Status
 - [ ] Complete, tested, ready to review and merge
 - [ ] Complete, but not tested (may need new tests)
 - [ ] Incomplete/work-in-progress, PR is for discussion/feedback
How Has This Been Tested?
Checklist
coauthors:
 - Dalton Walker
 - Michael Williams

---

## instructions on how to test

### test status accept or decline 

1. Create a main account (To create a project) 
1. then create a trade for the project. 
1. After that create a secondary account
1. apply to the main accounts project you created.
1. Then sign back in to your main account
1. go to dashboard to approve or decline the applicant.
1. verify the buttons disappear once you approve or decline applicant
1. verify status in top left corner changes to status you chose.
